xwiki-pro-macros-ui/src/main/resources/XWiki/Macros/Team.xml

```diff
@@ -851,6 +851,14 @@
     #end
   #end
   {{html wiki=false clean=false}}
+
+  #set($escapedSize = $escapetool.xml($size))
+        &lt;style&gt;
```

Comment:
It would be helpful to add a short comment explaining why this is needed. 
It makes sense in the context of this PR, but it might not be that obvious when someone else looks at the code